packages/passport/sdk/src/config/config.ts

```diff
@@ -62,6 +65,7 @@ export class PassportConfiguration {
     this.oidcConfiguration = oidcConfiguration;
     this.baseConfig = baseConfig;
     this.crossSdkBridgeEnabled = crossSdkBridgeEnabled || false;
+    this.announceProvider = announceProvider ?? true;
```

Comment:
Hey Charlie, is there any situation in which a consumer _wouldn't_ wan't Passport to announce itself? What's the use case here?

Reply:
Hey Hayden, it's a good question. The only use case I thought here is that some DApps may be listening for announced providers and showing them in a list. I imagine they would allowlist wallets they support but this option gives a little more granular control about how you are using the SDK and how you might want to connect to it.